src/components/post/PostHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Header, BackButton, Title } from './PostHeader.styled';
+
+interface PostHeaderProps {
+    title: string;
+    onBackClick?: () => void;
+}
+
+const PostHeader: React.FC<PostHeaderProps> = ({ title, onBackClick }) => {
+    const navigate = useNavigate();
+
+    const handleBackClick = () => {
+        if (onBackClick) {
+            onBackClick();
+        } else {
+            navigate(-1);
+        }
+    };
+
+    return (
+        <Header>
```

Comment:
```suggestion
        <S.Header>
            <S.BackButton onClick={handleBackClick} />
            <S.Title>{title}</S.Title>
        </S.Header>
```

이런 식으로 위에 import한 키워드(`S`)로 점 표기법(.) 사용해 작성하면 됩니다!